github-actions/register-data-assets/README.md

```diff
@@ -118,75 +121,48 @@ Options:
     -ph, --proxy-host TEXT        The host string of the database instance
                                   that serves as the proxy for the production
                                   database. This is the  database that Gable
-                                  will connect to when registering tables as
-                                  data assets instead of connecting to your
-                                  production database.  For example: in your
```

Comment:
any reason we are removing this example?

Reply:
Removed from the CLI help text in [this commit](https://github.com/gabledata/product/commit/320ef8eb52958cd9a91cf03c20b168482d40c23d#diff-18c0467c162601f257106723300c5f902270c4cc72e479778d42d06fd371b8d4L198)